en/docs/design/api-security/oauth2/token-revocation.md

```diff
@@ -61,4 +61,11 @@ The parameters required to invoke the following API are as follows:
             Transfer-Encoding: chunked
             
         ```
+    
+!!! note "Revoking access tokens obtained with a JWT grant"
```

Comment:
This note was removed from docs with PR https://github.com/wso2/docs-apim/pull/7782. Do we need to add it back? Does it work after changing the token revocation endpoint?

Reply:
This should be removed, seems like it was added back to 4.3.0 since the branches were out of sync. Removed from Fixed with https://github.com/wso2/docs-apim/pull/7925/commits/1945f39839794d27ede90b0d7e111d2908c219bb